dubbo-common/src/main/java/org/apache/dubbo/common/threadpool/serial/SerializingExecutor.java

```diff
@@ -26,14 +26,13 @@
 import org.apache.dubbo.common.threadlocal.InternalThreadLocalMap;
 
 import static org.apache.dubbo.common.constants.LoggerCodeConstants.COMMON_ERROR_RUN_THREAD_TASK;
-import static org.apache.dubbo.common.utils.ExecutorUtil.isShutdown;
 
 /**
  * Executor ensuring that all {@link Runnable} tasks submitted are executed in order
  * using the provided {@link Executor}, and serially such that no two will ever be
  * running at the same time.
  */
-public final class SerializingExecutor implements Executor, Runnable {
+public class SerializingExecutor implements Executor, Runnable {
```

Comment:
Change to abstract class would be better?

Reply:
It maybe used by other place, for example, somewhere need throw reject exception when submit failed.

Reply:
If in Dubbo we did not use in other place, just replace it. This is the internal code.

Reply:
Ok, I will change it.